src/rules/declaration-block-no-duplicate-properties/index.js

```diff
@@ -61,6 +67,31 @@ export default function (on, options) {
         const indexDuplicate = decls.indexOf(prop.toLowerCase())
 
         if (indexDuplicate !== -1) {
+
+          if (optionsHaveIgnored(options, "consecutive-duplicates-with-different-values")) {
```

Comment:
The CI error is caused by this, `optionsHaveIgnored` was just removed in #1765. The updated version should be:
```
optionsMatches(options, "ignore", "consecutive-duplicates-with-different-values")